src/midiEditorCommands.cpp

```diff
@@ -2292,6 +2293,94 @@ void postToggleFunctionKeysAsStepInput(int command) {
 	}
 }
 
+void postMidiToggleMute(int command) {
+	HWND editor = MIDIEditor_GetActive();
+	MediaItem_Take* take = MIDIEditor_GetTake(editor);
+	int eventCount = countSelectedEvents(take);
+	if (eventCount == 0) {
+		return;
+	}
+	// Get selected notes.
+	vector<MidiNote> selectedNotes = getSelectedNotes(take);
+	// Get selected CCs.
+	vector<MidiControlChange> selectedCCs = getSelectedCCs(take);
+	int noteCount = selectedNotes.size();
+	int CCCount = selectedCCs.size();
+	ostringstream s;
+	if (CCCount == 0) { // If only notes are selected
+		auto chord = findChord(take, 0, {
+			true,  // start
+			true,  // end
+			true,  // channel
+			true,  // pitch
+			true,  // velocity
+			false,  // selected
+			true  // muted
+		});
+		// Select and play the current chord.
```

Comment:
1. If I understand this correctly, this assumes that the selected notes are the current chord, but this might not be the case. It's possible to select notes all over the place. We probably shouldn't try to preview if any of the selected notes aren't within the current chord.
2. We aren't selecting anything here.